src/components/gameRules/gameRules.js

```diff
@@ -0,0 +1,39 @@
+import GAME_MODES from '../../consts/GAME_MODES';
+import './gameRules.css';
+
+const createTextContent = (gameMode, answer) => {
```

Comment:
what kind of text content it creates? 
function name indicates that it generate some message based on input - which it doesn't accept as parameter here

I think more meaningful name here could something like `gameDescriptionText`